configs/eslint-common-cypress.json

```diff
@@ -13,7 +13,7 @@
     },
     "overrides": [
         {
-            "files": ["./support/*.ts"],
+            "files": ["**/support/*.ts"],
```

Comment:
It was necessary to make it possible for other packages (components, stage and composer) to use it when the rules are defined in a file places in the root folder.